src/rapidhtml/app.py

```diff
@@ -1,13 +1,40 @@
 from __future__ import annotations
 
 import typing
-
+import inspect
 import uvicorn
 
 from starlette.applications import Starlette
+from pathlib import Path
 
 from rapidhtml.tags import Script
-from rapidhtml.routing import RapidHTMLRouter
+from rapidhtml.routing import RapidHTMLRouter, RapidHTMLWSEndpoint
+
+JS_RELOAD_SCRIPT = """           
+let active = false;
+sock = new WebSocket(`ws://${window.location.host}/live-reload`);
+sock.onopen = function (event) {
+    console.log(`connected`);
```

Comment:
Might as well be cute about it
```suggestion
    console.log(`Connected to the RapidHTML development server!`);
```